### ISSUE TITLE
Fails to find Texts (UTF_8)

### ISSUE THREAD

**Issue by Kyn0x** [Anonym0usWork1221/android-memorytool#2]
# **Describe the bug**
Hi! Attempting an `UTF_8` searches on version `0.3`; fails on Encoding. Performing another `DataTypes` is working but it's not for text related searches. Using a provided template also fails, the console omits the following:
```
Traceback (most recent call last):
  File "/data/media/0/Pictures/test.py", line 6, in <module>
    values = tool.read_value('Traffic')
  File "/data/data/com.termux/files/usr/lib/python3.10/site-packages/androidMemoryTool/androidMemoryTool.py", line 46, in read_value
    self._SearchAndReadObject.search_and_read_text(proc_id, maps_addr, value_to_read)
  File "/data/data/com.termux/files/usr/lib/python3.10/site-packages/androidMemoryTool/search_and_readers.py", line 122, in search_and_read_text
    byte_strings = mem_file.read(total_bytes)
OSError: [Errno 5] I/O error
```

# **To Reproduce**
A code snippet to reproduce the same behavior:
```python
tool = AndroidMemoryTool(PKG="com.emanuelef.remote_capture", TYPE=AndroidMemoryTool.DataTypes.UTF_8, SPEED_MODE=False, WORKERS=55,
                        pMAP=AndroidMemoryTool.PMAP(ALL=True))
values = tool.read_value('Traffic')
founded_values = values[1]
print(founded_values)
```

# **Expected behavior**
Performing searches with another DataTypes (`DWORD`) returns expected number of values exist. Using the provided template:
```
2062
['0x25b00c98', '0x25b02d9c', '0x25b049d0',....]
```

# **Screenshots**
[image](https://user-images.githubusercontent.com/116417953/197320495-16af89b0-0f6f-45c0-a7e9-84cef866a75e.png)

# **Specification**
 - Device: LDPlayer 9
 - OS: Android 9
 - Version: androidMemoryTool `0.3`

# **Additional context**
**I/O Error**
Running a workable searches (**DWORD**) still also omits `I/O Error` exception, although the result is working fine. Is this a normal behaviour that should be ignored?
[image](https://user-images.githubusercontent.com/116417953/197320517-a62f9cbb-fa5c-4bbd-a7b2-9d8c64b8f2c8.png)

**Documentation**
I hope there's more documentation or examples for each DataTypes. Especially for recent `0.3`, this is a bummer with such a great tool.

**Comment by Anonym0usWork1221** [comment]
**OSError: [Errno 5] I/O error**
-> Has nothing to do with tool
-> Only occurs in emulators, linux in VB, online servers
-> This is the error of OS.

As the tool read and write memory files. Some emulators doesn't allow some section like in your case UTF-8 to read or write. So, it is actually not a bug in tool may be in future I added emulator support properly.

**Comment by Anonym0usWork1221** [comment]
Fixed in android MemoryTool version 0.5
